server/src/routes/router.js

```diff
@@ -1,16 +1,30 @@
 const router = require('express').Router();
 
+const userController = require('../controllers/userController');
 const wishlistController = require('../controllers/wishlistController');
 const watchedlistController = require('../controllers/watchedlistController');
 
-// router.get('/wishlist', wishlistController.getAll);
-// router.get('/wishlist/:movieId', wishlistController.getOne);
-// router.post('/wishlist/:movieId', wishlistController.postOne);
-// router.delete('/wishlist/:movieId', wishlistController.deleteOne);
+router
+  .route('/wishlist')
+  .get(wishlistController.getAll)
+  .post(wishlistController.postOne);
+
+  router.delete('/wishlist/:movieId', wishlistController.removeOne);
+
+
+router
+  .route('/watched/')
+  .get(watchedlistController.getAll)
+  .post(watchedlistController.postOne);
+
+router.delete('/watched/:movieId', watchedlistController.removeOne);
+
+router.post('/user', userController.postOne);
+
+router
+  .route('/user/:userId')
+  .get(userController.getOne)
+  .delete(userController.removeOne);
```

Comment:
I think it would be nice to split the routes in different files as this file could get really big. We could have something like this:
routes
  |- watchedRouter.js
  |- wishlistRouter.js
  |- userRouter.js
  ...

And then in the server:
app.use('/wishlist', wishlist.Router) 
app.use('/watched', watched.Router)